assets/src/edit-story/components/inspector/design/designInspector.js

```diff
@@ -40,9 +42,15 @@ const TopPanels = styled.div`
 const BottomPanels = styled.div``;
 
 function DesignInspector() {
+  const resetStoryAnimationState = useStory(({ actions }) => () =>
+    actions.updateAnimationState({
+      animationState: STORY_ANIMATION_STATE.RESET,
+    })
+  );
```

Comment:
I think this might cause a rerender every time since it will be checking equality on a new function. Think I'm gonna have to just pull `updateAnimationState` out of here and wrap it in a `useCallback` to get the desired behavior